src/rsysreqs/scripts/scripts_ubuntu.go

```diff
@@ -0,0 +1,14 @@
+package scripts
+
+import "fmt"
+
+type ubuntuScriptGenerator struct{}
+
+func (g ubuntuScriptGenerator) InstallScripts(packages []string) []string {
+	var scripts []string
+	for _, pkg := range packages {
+		script := fmt.Sprintf("apt-get install -y %s", pkg)
+		scripts = append(scripts, script)
+	}
+	return scripts
+}
```

Comment:
Maybe these templates could be defined in external data, but I thought this was simpler for the first go